chemprop/data/data.py

```diff
@@ -58,6 +62,16 @@ def __init__(self,
             replace_token = 0
             self.features = np.where(np.isnan(self.features), replace_token, self.features)
 
+        # Fix nans in atom_descriptors
+        if self.atom_descriptors is not None:
+            replace_token = 0
+            self.atom_descriptors = np.where(np.isnan(self.atom_descriptors), replace_token, self.atom_descriptors)
+
+        # Fix nans in atom_features
+        if self.atom_features is not None:
+            replace_token = 0
+            self.atom_features = np.where(np.isnan(self.atom_features), replace_token, self.atom_features)
```

Comment:
Minor suggestion, but maybe you could move `replace_token = 0` before the three `if` statements so that it only appears once rather than three times over.